code/modules/mob/living/emote.dm

```diff
@@ -8,25 +8,22 @@
 
 	var/param = null
 
-	if (findtext(act, "-", 1, null))
+	if (findtext(act, "-", 1, null)) //Removes dashes for npcs "EMOTE-PLAYERNAME" or something like that, I ain't no AI coder. It's not for players. -Sum99

```

Comment:
It's actually have you have an emote with arguments.

*point-Lindsay Donk
for example